sound/soc/intel/boards/sof_da7219.c

```diff
@@ -180,6 +180,19 @@ static void da7219_codec_exit(struct snd_soc_pcm_runtime *rtd)
 
 static int card_late_probe(struct snd_soc_card *card)
 {
+	struct sof_card_private *ctx = snd_soc_card_get_drvdata(card);
```

Comment:
commit message, please fix the checkpatch error on the SHA1 value

Reply:
fixed